### PR TITLE
UTY-2626: Update spatialos json

### PR DESCRIPTION
**Pull request by BryanJY-Wong**
#### Description
Update to 14.6.1

#### Tests
None

#### Documentation
- [x] Suggested Edit on docs

- [x] **Did you remember a changelog entry?**

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 ### Changed
 
 - Moved Gdk Tools Configuration to the Unity "Project Settings" window under `Spatial OS`. [#1408](https://github.com/spatialos/gdk-for-unity/pull/1408)
-- `spatialos.json` sdk and standard library version updated to `14.6.1`
+- `spatialos.json` sdk and standard library version updated to `14.6.1`. [#1422](https://github.com/spatialos/gdk-for-unity/pull/1422)
```
Comment (paulbalaji) [inline CHANGELOG.md:28]: Don't think this needs a changelog as it's purely a cosmetic thing to make it look like we're not lagging behind versions

**Comment by zeroZshadow** [comment]
Is the standard_library dependency still needed? Or can we remove this and have 1 less version to keep up to date.

File: CHANGELOG.md (hunk 1 above)
Comment (paulbalaji) [reply CHANGELOG.md:28]: same comment applies to corresponding FPS/Blank PRs

File: spatialos.json [hunk 2]
```diff
@@ -1,8 +1,5 @@
 {
     "name": "unity_gdk",
     "project_version": "0.0.1",
-    "sdk_version": "14.5.0",
-    "dependencies": [
-        {"name": "standard_library", "version": "14.5.0"}
-    ]
+    "sdk_version": "14.6.1",
```
Comment (paulbalaji) [inline spatialos.json:4]: nit
```suggestion
    "sdk_version": "14.6.1"
```

**Pull request merged**